lib/recurly/hosted-fields.js

```diff
@@ -29,13 +30,55 @@ export class HostedFields extends Emitter {
     this.errors = [];
     this.initQueue = [];
     this.readyState = 0;
+    this.onTabNext = this.onTabNext.bind(this)
+    this.onTabPrevious = this.onTabPrevious.bind(this)
     this.configure(options);
     this.inject();
     this.on('hostedField:state:change', this.update.bind(this));
     this.on('bus:added', bus => {
       this.bus = bus;
       this.fields.forEach(hf => bus.add(hf));
     });
+
+    this.on('hostedField:tab:previous',this.onTabPrevious)
+    this.on('hostedField:tab:next',this.onTabNext)
+  }
+
+  onTabNext(body) {
+    let type = body.type
+    let field = this.fields.find(function(f){ return f.type === type })
+    let allFields = document.querySelectorAll("[data-recurly]")
```

Comment:
What do you think the best way to do that is? The previous and next ones that aren't hosted fields are pretty difficult to find, or at least I don't see an easy way to figure that out. If the elements have a taborder attribute it's not too difficult, but I'm not sure exactly how that would work in practice. Would clients put taborder on like the parent div of hosted fields also?  I think that would probably mess up the way I'm using a fake input before the first hosted field (outside the iframe) to have something to tab into.